### PR TITLE
Feat(crd): init model job crd when run the controller

### PR DESCRIPTION
**Pull request by FogDong**
**What this PR does / why we need it**:

Feat(crd): init model job crd when run the controller

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @simon-cj @judgeeeeee 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by judgeeeeee** [comment]
LGTM

File: cmd/modeljob-operator/app/server.go [hunk 1]
```diff
@@ -84,3 +109,109 @@ func Run(opt *options.ServerOption) error {
...
+	if err != nil {
+		return err
+	}
+	for _, crd := range crds {
+		_, err = client.ApiextensionsV1beta1().CustomResourceDefinitions().Create(crd)
+		if err != nil {
+			setupLog.Error(err, "unable to ensure crds")
+			if !errors.IsAlreadyExists(err) {
+				return err
+			}
```
Comment (simon-cj) [inline cmd/modeljob-operator/app/server.go:212]:
```suggestion
			if !errors.IsAlreadyExists(err) {
				return err
			}
			setupLog.Error(err, "unable to ensure crds")
```

Comment (judgeeeeee) [reply cmd/modeljob-operator/app/server.go:212]: ？

**Pull request closed without merging**